Sources/SolToBoogie/ProcedureTranslator.cs

```diff
@@ -122,6 +122,40 @@ public override bool Visit(EnumDefinition node)
             return false;
         }
 
+        private BoogieCallCmd GenerateBoogieCallCmd(TypeDescription type, BoogieExpr value, string name)
+        {

```

Comment:
We don't want to generate values when the expression is an array.

Reply:
We discussed a check that will check for IsDynamic and Static array and skip the callCmd generation.